README.adoc

```diff
@@ -99,6 +99,17 @@ config.assets.precompile += %w( rails_admin/rails_admin.css rails_admin/rails_ad
 
 Keep configuration that's applicable to all environments in the `config/application.rb` file.
 
+=== Load Rails Config Defaults [[config-defaults]]
+
+Load Rails' configuration defaults in the `config/application.rb` corresponding to the used Rails version.
```

Comment:
I think a little more guidance/explanation can be given here:
```suggestion
When upgrading to a newer Rails version, your application's configuration setting will remain on the previous version. To take advantage of the latest recommended Rails practices, the `config.load_defaults` setting should match your Rails version.
```

Reply:
Significantly better said, thank you!